.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
```

Comment:
neutrino use  airbnb as eslint config , see neutrinorc.js . So use the same config here